lib/resin.coffee

```diff
@@ -219,3 +224,64 @@ module.exports = getSdk = (opts = {}) ->
 	sdk.errors = errors
 
 	return sdk
+
+###*
+# @summary Set shared default options
+# @name setSharedOptions
+# @public
+# @function
+# @memberof resin
+#
+# @description
+# Set options that are used by calls to `resin.fromSharedOptions()`.
+# The options accepted are the same as those used in the main SDK factory function.
```

Comment:
Should document here that this needs to be called as soon as possible during app startup, and definitely before any calls to `fromSharedOptions`